### ISSUE TITLE
livepeer_cli taking more than requested for reserve when calling fundDepositAndReserve

### ISSUE THREAD

**Issue by chrishobcroft** [livepeer/go-livepeer#1211]
**Describe the bug**

When depositing Broadcasting funds via the livepeer_cli, it seems to take more for the reserve than was requested.

When invoking `12. Invoke "deposit broadcasting funds" (ETH)`, I asked to deposit `0.5 ETH` for deposit and `0.1 ETH` for reserve.

The amount taken for reserve was `0.100000000000000005 ETH`

**To Reproduce**

Steps to reproduce:

1. Run a node in Broadcaster mode
2. Run livepeer_cli
3. Choose the command: `12. Invoke "deposit broadcasting funds" (ETH)`
4. Enter `0.5` for deposit and `0.1` for reserve and wait for transaction to confirm
5. Choose the command: `1. Get node status`
6. See that the amount taken for reserve was `0.100000000000000005 ETH`

**Expected behavior**

I expect it to take exactly the same amount that I instructed it to take.

**Screenshots**

Amount requested for deposit / reserve

![Screenshot from 2019-11-22 21-19-08](https://user-images.githubusercontent.com/2212651/69461819-595cb400-0d6f-11ea-9c52-4ea15fc1063c.png)

Amounts actually requested by the process for deposit and reserve:

![Screenshot from 2019-11-22 21-19-01](https://user-images.githubusercontent.com/2212651/69461818-595cb400-0d6f-11ea-82ae-2bc878ecf062.png)

Transaction called by the node (incidentally, the values are not logged in the console - perhaps they should / could be?)

![Screenshot from 2019-11-22 21-20-11](https://user-images.githubusercontent.com/2212651/69461820-595cb400-0d6f-11ea-9be5-b7ea3ea0718c.png)

Amount shown on Etherscan:

![Screenshot from 2019-11-22 21-31-51](https://user-images.githubusercontent.com/2212651/69461902-8c06ac80-0d6f-11ea-8e28-00c2a38a95d1.png)

And amount requested for reserve, in hexadecimal

![Screenshot from 2019-11-22 21-24-29](https://user-images.githubusercontent.com/2212651/69461821-595cb400-0d6f-11ea-93b0-d70a951c01dd.png)

And amount requested for reserve converted from hex to decimal... showing that the process was calling the function with the incorrect input.

![Screenshot from 2019-11-22 21-24-41](https://user-images.githubusercontent.com/2212651/69461822-595cb400-0d6f-11ea-9047-670dc9d733ef.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 19.04
```
chrishobcroft@thinkpad:~$ ./livepeer -version
Livepeer Node Version: 0.5.0
Golang runtime version: gc go1.13
Architecture: amd64
Operating system: linux
```

**Comment by kyriediculous** [comment]
Thanks for reporting Chris as always ! I'll be tackling some of these issues in the coming days.

> Transaction called by the node (incidentally, the values are not logged in the console - perhaps they should / could be?)

Yes you're totally right, and this functionality used to exists because we used a fork of `go-ethereum` in which we included this logging of transaction params ourselves. Part of recent updates make it so that we no longer depend on a fork of `go-ethereum` (which I think is a good thing !). I've spent some time looking at how we can include transaction params without maintaining our own `go-ethereum` fork but I have to break my head over it a little more.
